Fix depreciated tr1 call in rebuilder_test.cpp

reiniscimurs opened the pull request:
File calls fro tr1 in std which is depreciated. Removing the tr1 call seems to fix the "error: ‘tr1’ is not a member of ‘std’"

hufman commented:
Some of the songs from Shantae: Half Genie Hero seem to not have loop points, the game seems to loop the entire sample. I'm not sure whether the FSB stores the decision to single-shot or loop the entire sample.

tmiasko commented on src/fsb/vorbis/rebuilder.cpp:
```diff
@@ -193,9 +195,19 @@ void rebuilder::rebuild_id_header(
+  char s_loop_start[20];
+  char s_loop_end[20];
+  snprintf(s_loop_start, 20, "%i", loop_start);
+  snprintf(s_loop_end, 20, "%i", loop_end);
```
Could you use `PRIu32` macro from `<cinttypes>` header instead of `%i`. Or even
better, just convert it to string with `to_string` followed by `c_str` to get
null-terminated C string?

hufman commented:
Happy holidays! I updated this change to use to_string, which feels much more elegant.

tmiasko commented:
Merged. Happy holidays!

reiniscimurs committed:
Fix depreciated tr1 call in rebuilder_test.cpp

File calls fro tr1 in std which is depreciated. Removing the tr1 call seems to fix the "error: ‘tr1’ is not a member of ‘std’"
